src/js/components/StatusBar.js

```diff
@@ -11,22 +11,74 @@ class StatusBar extends React.Component {
     if (max === 0) {
       return null;
     }
+    let indexesLessThanThreshold = [];
+    let maxIndex = 0;
+    let maxSize = 0;
+    // Size of largest portion to give to smallest portion. We have a max of 5
+    // status types so we want to keep this growth ratio relatively small.
+    let growRatio = 0.07;
+    // Grow anything below this threshold by stealing from the biggest portion
+    let threshold = 7;
+
+    let barSizes = data.map(function (status, index) {
+      let {value} = status;
+
+      let relativeSize = value / max * 100;
+
+      if (relativeSize > 0 && relativeSize < threshold) {
+        indexesLessThanThreshold.push(index);
+      }
+
+      return relativeSize;
+    });
+
+    function stealPortion() {
+      if (indexesLessThanThreshold.length) {
+        // Try to steal some width from the largest portion
+        // and give it to the first portion in indexesLessThanThreshold
+        maxSize = 0;
+        maxIndex = 0;
```

Comment:
Can you prefix these with `let` I don't think we need to be using the ones from the closure.